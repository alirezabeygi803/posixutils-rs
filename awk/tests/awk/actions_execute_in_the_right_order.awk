END { print "e1" }

BEGIN { print "b1" }

1 { print 1 }

BEGIN { print "b2" }

1 { print 2 }

END { print "e2" }

1 { print 3 }

BEGIN { print "b3" }

END { print "e3" }
