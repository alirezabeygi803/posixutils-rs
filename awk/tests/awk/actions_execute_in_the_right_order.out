b1
b2
b3
1
2
3
1
2
3
1
2
3
1
2
3
1
2
3
1
2
3
1
2
3
e1
e2
e3
